penguin/api_client.py

```
@@ -39,14 +39,10 @@
 
 import asyncio
 import logging
-from pathlib import Path
-from typing import Any, Dict, List, Optional, Sequence, Union, Callable, AsyncGenerator
+from typing import Any, Dict, List, Optional, Callable, AsyncGenerator
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)